tutorials/Getting_Started/README.md

```diff
@@ -106,37 +104,53 @@ import androidx.compose.ui.Modifier
 import androidx.compose.ui.unit.IntSize
 import androidx.compose.ui.unit.dp
 
-fun main() = Window(title = "Compose for Desktop", size = IntSize(300, 300)) {
-    val count = remember { mutableStateOf(0) }
+fun main() = Window(
+        title = "Compose for Desktop", 
+        size = IntSize(300, 300)
+        ) {
+    val count by remember { mutableStateOf(0) }
     MaterialTheme {
-        Column(Modifier.fillMaxSize(), Arrangement.spacedBy(5.dp)) {
-            Button(modifier = Modifier.align(Alignment.CenterHorizontally),
+        Column(
+            modifier = Modifier.fillMaxSize(), 
+            // spacedBy() sets the distance between each Composable
+            Arrangement.spacedBy(5.dp)
+        ) {
+            Button(
+                modifier = Modifier.align(Alignment.CenterHorizontally),
                    onClick = {
+                       // Becareful using .value++, it can cause un needed Recomposes
                        count.value++
-                   }) {
-                Text(if (count.value == 0) "Hello World" else "Clicked ${count.value}!")
+                   }
+            ) {
+                Text(
+                    if (count.value == 0) 
+                        "Hello World" 
+                    else 
+                        "Clicked ${count.value}!")
             }
-            Button(modifier = Modifier.align(Alignment.CenterHorizontally),
+            Button(
+                modifier = Modifier.align(Alignment.CenterHorizontally),
                    onClick = {
                        count.value = 0
-                    }) {
+                    }
+            ) {
                 Text("Reset")
             }
         }
     }
 }
 ```
-## Running the project
+## Running The Project
 
 Open `build.gradle.kts` as a project in IntelliJ IDEA.
 
 ![New project](screen1.png)
 
 After you download the Compose for Desktop dependencies from the Maven repositories your new project is ready
-to go. Open the Gradle toolbar on the right, and select `sample/Tasks/compose desktop/run`.
+to go. Open the Gradle toolbar on the right and select `sample/Tasks/compose desktop/run`.
 The first run may take some time, but afterwards the following dialog will be shown:
 
 ![Application running](screen2.gif)
 
-You can click on the button several times and see that the application reacts and
+Finally you can click on the button several times and see that the application reacts and
```

Comment:
And finally, you can click on the button several times and see that the application reacts and
updates the UI.

Reply:
It is not proper grammar to start a sentence with And.